src/api/stats/getChainTvl.js

```diff
@@ -38,11 +46,36 @@ const getChainTvl = async chain => {
     tvls[chainId] = { ...tvls[chainId], ...item };
   }
 
-  if (chain.governancePool) {
-    let governanceTvl = await getGovernanceTvl(chainId, chain.governancePool);
-    tvls[chainId] = { ...tvls[chainId], ...governanceTvl };
-  } else {
-    console.log('chainId', chainId, 'no gov pool');
+  for (let i = 0; i < govVaults.length; i++) {
+    const govVault = govVaults[i];
+
+    let tokenPrice = 0;
+    const { oracle, oracleId, tokenDecimals, excluded } = govVault;
+    try {
+      tokenPrice = await fetchPrice({ oracle: oracle, id: oracleId });
+      // tokenPrice = 25;
+    } catch (e) {
+      console.error('getGovernanceTvl fetchPrice', chainId, oracle, oracleId, e);
+    }
+
+    let balance = govVaultBalances[i].shiftedBy(-tokenDecimals);
+    let excludedTvl = new BigNumber(0);
+
+    if (excluded) {
+      const vault = getVaultByID(excluded);
+      if (vault && vault.status === 'active') {
+        excludedTvl = new BigNumber(tvls[chainId][excluded] || 0);
+      }
+    }
+    const tvl = balance.times(tokenPrice);
+
+    let item = { [govVault.id]: 0 };
+
+    if (!tvl.isNaN()) {
+      item = { [govVault.id]: Number(tvl.minus(excludedTvl).toFixed(2)) };
+    }
+
+    tvls[chainId] = { ...tvls[chainId], ...item };
```

Comment:
This block of code is super similar to the one above, it can be either merged with some conditions or refactored a bit